docs/reference/ssl/kubernetes-with-cert-manager.md

```diff
@@ -0,0 +1,197 @@
+# SSL on Kubernetes
+
+The quickest way to get automated SSL/TLS certificates for your project is to use [cert-manager][cert-manager]. In this tutorial, we will deploy OpenFaaS using the [Helm Chart][openfaas-helm], [cert-manager][cert-manager], and [nginx-ingress][nginx-ingress]
+
+**RBAC** We will assume that your cluster is configured for [RBAC][k8s-rbac].
+
+## Configure Helm and Tiller
+
+First install Helm and the Tiller [following the instructions provided by Helm][helm-install]
+
+## Create namespaces
+
+```sh
```

Comment:
We've already done this, plus they should not be created this way. There is a YAML file which contains special labels - see the Helm Chart readme.